op-node/node/config.go

```diff
@@ -64,6 +64,16 @@ type Config struct {
 
 	// [OPTIONAL] The reth DB path to read receipts from
 	RethDBPath string
+
+	// Conductor is used to determine this node is the leader sequencer.
+	ConductorEnabled    bool          `json:"conductor_enabled"`
+	ConductorAddr       string        `json:"conductor_addr"`
+	ConductorPort       int           `json:"conductor_port"`
+	ConductorRpcTimeout time.Duration `json:"conductor_rpc_timeout"`
```

Comment:
Is there a reason we're defining json names for this when the other fields don't?

Reply:
no this is a typo, originally had this in the driver config which does have json fields. removed.